src/CountryData.h

```diff
@@ -8,64 +8,72 @@
-        @param web::json::value& json - A JSON value as an object returned from the API request.
-        @return CountryData - A new CountryData object parsed from the JSON object returned from the API.
-        */
-        static CountryData parseJson(web::json::value& json);
-
-        /*
-        Gets a new JSON value AS AN ARRAY from the cpprestsdk returned from the API request and
-        parse every JSON object inside the array to a new CountryData object, adding it to a vector.
-        @param web::json::value& jsonArray - A JSON value as an array returned from the API request.
-        @return std::vector<CountryData> - A new vector of CountryData objects parsed from the
-        JSON array returned from the API.
-        */
-        static std::map<std::string, CountryData> parseJsonArray(web::json::value& jsonArray);
-
-        /* Identifies that this object node was added to the OPC UA address space. */
-        void setIsInitialized(const bool initialized);
-        void setLocations(const std::map<std::string, LocationData>& allLocations);
-
-        std::string getName() const { return name; }
-        std::string getCode() const { return code; }
-        uint32_t getCitiesNumber() const { return citiesNumber; }
-        uint32_t getLocationsNumber() const { return locationsNumber; }
-        bool getIsInitialized() const { return isInitialized; }
-        std::map<std::string, LocationData>& getLocations() { return locations; }
-
-        // Constants Representing the string(key) of the pair string:value of JSON objects.
-        static const utility::string_t KEY_NAME; // Key country's name in the JSON result.
-        static const utility::string_t KEY_CODE; // Key country's code in the JSON result.
-        static const utility::string_t KEY_CITIES; // Key country's city in the JSON result.
-        static const utility::string_t KEY_LOCATIONS; // Key country's location in the JSON result.
-
-        // C-Style String representing the browse/display name of the variables nodes in OPC UA.
-        static char COUNTRIES_FOLDER_NODE_ID[];
-        static char BROWSE_NAME[];
-        static char BROWSE_CODE[];
-        static char BROWSE_CITIES_NUMBER[];
-        static char BROWSE_LOCATIONS_NUMBER[];
-
-    private:
-        std::string name;
-        std::string code;
-        uint32_t citiesNumber;
-        uint32_t locationsNumber;
-        bool isInitialized;
-        std::map<std::string, LocationData> locations;
-    };
+    static CountryData parseJson(web::json::value& json);
+
+    /*
+    Gets JSON value AS AN ARRAY and parses every JSON object inside into CountryData object, adding it to a map if it has correct values.
+    Correct values - name & country code are not empty.
+
+    @param web::json::value& jsonArray - array of JSON values returned from Open AQ API request.
+    @return std::map<std::string, CountryData> - map of CountryData objects with two letter country code as a key.
```

Comment:
No need to repeat types in doc-comments, only names for variables should be used.